src/lib/lore.sh

```diff
@@ -423,8 +423,7 @@ function process_patchsets()
 
       if is_introduction_patch "$patch_url"; then
         # Process each patch in parallel
-        thread_for_process_patch "$PATCHSETS_PROCESSED" "$shared_dir_for_parallelism" \
-          "$processed_patchset" "$patch_url" "$patch_title" &
+        thread_for_process_patch "$PATCHSETS_PROCESSED" "$shared_dir_for_parallelism" "$processed_patchset" "$patch_url" "$patch_title" &
```

Comment:
This solution sounds fragile... can you maybe expand your parser from the previous commit to check if the line ends with `\` and, based on that, compose with the following line?

Reply:
Done